src/guidellm/request/transformers.py

```diff
@@ -1,9 +1,9 @@
 from pathlib import Path
 from typing import Optional, Union
 
-from datasets import (  # type: ignore  # noqa: PGH003
+from datasets import (
     Dataset,
-    DatasetDict,
+    DatasetDict,  # type: ignore  # noqa: PGH003
```

Comment:
With the mypy overrides added in, the type: ignore should be able to be removed now, correct?

Reply:
Oh, yeah!